Add flags to start script (#23)
exercism/development-environment: README.md

### Updating everything

The first time `./bin/start` is run, it will download the latest versions of all the Exercism components for you to use.

> When you want to force updating to the latest version of those components — which we advise doing regularly — run:

Reviewer: ```suggestion
If you want to download and run the latest version of those components — which we advise doing regularly — run:
```

docker-compose pull
```

You can also pass the `--pull` argument to `./bin/start`:

```bash
./bin/start --pull
```

> 

Reviewer: I suggest replacing the `docker-compose pull` command with this rather than having it as an addition.

You can then run `./bin/start` as normal.

Alternatively, you can skip the `docker-compose build` step and pass the `--build` flag to `./bin/start`:

```bash
./bin/start --build

> ```

Reviewer: As above, should we remove the `docker-compose --build ...` step as a seperate suggestion and just have this?
